### PR TITLE
Add image streams for prescriptions

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/adviser/issues/1908

### CODE REVIEW

**Comment by fridex** [comment]
/assign @harshad16 

I tried to double check, but might be some copy-pasta issue here.

File: prescriptions/application.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
```
Comment (harshad16) [inline prescriptions/application.yaml:1]: we can drop this file.

File: prescriptions/overlays/ocp4-stage/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: thoth-backend-stage
```
Comment (harshad16) [inline prescriptions/overlays/ocp4-stage/kustomization.yaml:3]: lets drop this line.

**Review by harshad16 — COMMENTED** [review]
awesome :tada: 
lgtm, just two minor comments.
thanks :100:

File: prescriptions/overlays/ocp4-stage/kustomization.yaml (hunk 2 above)
Comment (fridex) [reply prescriptions/overlays/ocp4-stage/kustomization.yaml]: Adjusted in 42209449 👍🏻

File: prescriptions/application.yaml (hunk 1 above)
Comment (fridex) [reply prescriptions/application.yaml]: Adjusted in 42209449 👍🏻

**Comment by goern** [comment]
/lgtm
it requires an argocd app, right?

**Comment by fridex** [comment]
Adjusted based on suggestions @harshad16

**Pull request merged**